pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -508,8 +478,8 @@ func (n *NetworkPolicyController) createAppliedToGroup(npNsName string, pSel, nS
 // NetworkPolicyPeer object in NetworkPolicyRule. This function simply
 // creates the object without actually populating the PodAddresses as the
 // affected GroupMembers are calculated during sync process.
-func (n *NetworkPolicyController) createAddressGroup(namespace string, podSelector, nsSelector, eeSelector *metav1.LabelSelector) string {
-	groupSelector := toGroupSelector(namespace, podSelector, nsSelector, eeSelector)
+func (n *NetworkPolicyController) createAddressGroup(namespace string, podSelector, nsSelector, eeSelector *metav1.LabelSelector, nodeSelectors ...*metav1.LabelSelector) string {
```

Comment:
I don't think variadic function applies here. Even if we have more selectors in future, they will have different meaning and shouldn't be represented by "nodeSelectors".

Reply:
Updated.